src/app.js

```diff
@@ -1,21 +1,51 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs/promises');
+const { errorMessage } = require('./utils/errorMessage');
+
+const POPT = 3001;
+const EXPECTED_FOLDER = '/file';
+const DATA_PATH = './public';
+const DEFAULT_FILE = '/index.html';
+
+const server = http.createServer(async(req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+  const text = normalizedURL.pathname;
+
+  res.setHeader('Content-Type', 'application/json');
+
+  if (text.slice(0, EXPECTED_FOLDER.length) === EXPECTED_FOLDER) {
+    let path = text.slice(EXPECTED_FOLDER.length);
+
+    if (path === '' || path === '/') {
+      path = DEFAULT_FILE;
+    }
+
+    try {
+      const file = await fs.readFile(DATA_PATH + path);
+
+      res.statusCode = 200;
```

Comment:
wil be good create constants with status codes and reuse `200, 400`

Reply:
Thanks for it!